src/shared/path-lookup.c

```diff
@@ -111,7 +111,8 @@ static char** user_dirs(
                 const char *runtime_config,
                 const char *generator,
                 const char *generator_early,
-                const char *generator_late) {
+                const char *generator_late,
+                const char *transient) {
```

Comment:
So, `is-enabled` will return "static" for transient units if I'm reading the code right. Is this desired, or should we add another state to the table?